tfprotov5/tf5server/server.go

```diff
@@ -2,16 +2,27 @@ package tf5server
 
 import (
 	"context"
+	"errors"
+	"log"
+	"regexp"
+	"strings"
 	"sync"
 
-	"github.com/hashicorp/go-hclog"
-	"github.com/hashicorp/go-plugin"
 	"github.com/hashicorp/terraform-plugin-go/tfprotov5"
 	"github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/fromproto"
 	"github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5"
 	"github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/toproto"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/go-plugin"
+	"github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-log/tfsdklog"
+	tfaddr "github.com/hashicorp/terraform-registry-address"
 )
 
+const tflogSubsystemName = "proto"
```

Comment:
Open question: Do we want to differentiate between proto5 and proto6, so folks can debug which server is being used, in case it matters?

Reply:
Hot take: not in the subsystem name, we could as part of the trace logging, though. Included that.